client/packages/common/src/ui/forms/JsonForms/components/Array.tsx

```diff
@@ -111,6 +113,22 @@ const EnumArrayComponent: FC<EnumArrayControlCustomProps> = ({
         <Box flexBasis={FORM_INPUT_COLUMN_WIDTH}>
           <Autocomplete
             multiple
+            sx={{
+              '.rtl-w7osp1-MuiInputBase-root-MuiInput-root:before': {
+                border: 'none',
+                borderRadius: '8px',
+                height: '100%',
+                backgroundColor: theme.palette.background.drawer,
+             },
+              '.rtl-1fjcdrw-MuiButtonBase-root-MuiChip-root': {
+                backgroundColor: theme.palette.secondary.light,
+                height: 'inherit',
+                color: theme.typography.login.color,
```

Comment:
Login was the only typography in white... guess could have added another separate one for this 🤔